src/combat.cpp

```diff
@@ -1402,17 +1404,17 @@ void MagicField::onStepInField(Creature* creature)
 			bool harmfulField = true;
 
 			if (g_game.getWorldType() == WORLD_TYPE_NO_PVP || getTile()->hasFlag(TILESTATE_NOPVPZONE)) {
-				Creature* owner = g_game.getCreatureByID(ownerId);
+				auto owner = g_game.getCreatureByID(ownerId);
```

Comment:
Could this be const auto
```suggestion
				const auto owner = g_game.getCreatureByID(ownerId);
```